sdk/monitor/azure-monitor-query/azure/monitor/query/_helpers.py

```diff
@@ -41,13 +41,21 @@ def get_metrics_authentication_policy(
 
     raise TypeError("Unsupported credential")
 
-def process_error(exception):
-    raise_error = HttpResponseError
-    raise raise_error(message=exception.message, response=exception.response)
-
-def order_results(request_order, mapping, obj):
+def order_results(request_order, mapping, obj, err, allow_partial_errors=False):
```

Comment:
Do we want to put allow_partial_errors into kwargs?

Reply:
normally i would say yes - but here, i don't see the kwargs expanding anytime soon - plus this is an internal method, so we can change it if we think it's necessary